pylasu/model/model.py

```diff
@@ -95,13 +96,56 @@ def provides_nodes(decl_type):
         return isinstance(decl_type, type) and issubclass(decl_type, Node)
 
 
+def get_only_type_arg(decl_type):
```

Comment:
Would it make sense to use type annotations for both the parameter and the return type?

Reply:
I wouldn't know how to properly type them